docs/src/main/asciidoc/_admin-guide/Management_resources.adoc

```diff
@@ -123,17 +147,35 @@ details of one of them, via the CLI one would:
         "request-properties" => {"url" => {
             "type" => STRING,
             "description" => "The URL at which the deployment content is available for upload to the domain's or standalone server's deployment content repository.. Note that the URL must be accessible from the target of the operation (i.e. the Domain Controller or standalone server).",
+            "expressions-allowed" => false,
             "required" => true,
-            "min-length" => 1,
-            "nillable" => false
+            "nillable" => true,
+            "alternatives" => [
+                "input-stream-index",
+                "hash",
+                "bytes",
+                "path",
+                "relative-to",
+                "empty"
+            ],
```

Comment:
@soul2zimate I realize now when I filed https://issues.redhat.com/browse/WFCORE-6981 I was overly focused on the 'reply-properties' output. This nillable=true and 'alternatives' stuff is wrong as well, for the same basic reason -- the OperationDescription is reusing an AttributeDefinition that's meant for a different op.

Reply:
@soul2zimate I updated this bit to use 'nillable => false' and no alternatives, as that's the correct data and will be what's reported when https://issues.redhat.com/browse/WFCORE-7066 is fixed.